hack/operator-sdk-common.sh

```diff
@@ -3,11 +3,19 @@
 # shellcheck disable=SC2034
 # don't fail on unused variables - this is for sourcing
 
+
+# Align with installation instructions from:
+# https://sdk.operatorframework.io/docs/installation/
+OPERATOR_SDK_ARCH=$(uname -m)
+OPERATOR_SDK_OS=$(uname | awk '{print tolower($0)}')
```

Comment:
If we already have `OS_TYPE` do we need to do this? Looks like you could just do this in `common.sh` (including the hack below), you would only need to additionally update `shellcheck-test.sh`. Also, at this point we might want to define "constants" of `OS_TYPE_LINUX` and `OS_TYPE_DARWIN` (in all lowercase) for our expected values and compare against those everywhere. This sounds like it'd be a separate commit before introducing the operator-sdk hack on top of it.

Reply:
Fair enough!